src/Gaufrette/Adapter/AzureBlobStorage.php

```diff
@@ -403,4 +481,26 @@ private function guessContentType($content)
 
         return $fileInfo->buffer($content);
     }
+
+    /**
+     * @param string $key
+     *
+     * @return array
+     * @throws \InvalidArgumentException
+     */
+    private function tokenizeKey($key)
+    {
+        $containerName = $this->containerName;
+        if (true === $this->multiContainerMode) {
```

Comment:
I would rather invert the conditional ` true === $this->multiContainerMode` to early return a value when it's disabled. It would make it a bit more readable.